packages/php/blueprint/src/Importers/ImportSetSiteOptions.php

```diff
@@ -27,16 +49,23 @@ class ImportSetSiteOptions implements StepProcessor {
 	public function process( $schema ): StepProcessorResult {
 		$result = StepProcessorResult::success( SetSiteOptions::get_step_name() );
 		foreach ( $schema->options as $key => $value ) {
+			// Skip if the option should not be modified.
+			if ( in_array( $key, self::RESTRICTED_OPTIONS, true ) ) {
+				$result->add_warn( "Cannot modify '{$key}' option: restricted for security reasons." );
```

Comment:
```suggestion
				$result->add_warn( "Cannot modify '{$key}' option: Modifying is restricted for this key." );
```
I wonder if we should remove the "for security reasons" and just say something like the above.

Reply:
Sounds good! Thanks for suggesting.

Reply:
@chihsuan feel free to merge this, thanks for the updates.

Reply:
Thanks. I'll wait to merge this until after the base branch merges, which will keep the base branch clean and simplify the review process.